databases/week3/Part2-Queries.txt

```diff
@@ -0,0 +1,126 @@
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+--Inserting values
+--meal data
+
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Dosa', 'Indian rice and dal pancakes', 'Vesterbro', '2020-02-14 17:00:00', 25, 20, '2020-02-11 00:00:00');
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Jeera pulao', 'Indian rice with spices', 'Amager', '2020-02-14 17:00:00', 50, 25.99, '2020-02-11 00:00:00');
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Naan', 'Indian bread made in tandoor', 'Amager', '2020-02-14 17:00:00', 50, 100, '2020-02-11 00:00:00');
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Paneer butter masala', 'Spicy gravy made with Indian cottage cheese', 'Amager', '2020-02-14 17:00:00', 50, 25, '2020-02-11 00:00:00');
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Mango lassi', '', 'Valby', '2020-02-18 22:00:00', 50, 25, '2020-02-11 00:00:00');
+
+
+--reservation data
+
+insert into reservation (number_of_guests, meal_id, created_date) values (100, 2,'2020-02-11 12:41:00');
+insert into reservation (number_of_guests, meal_id, created_date) values (2, 5,'2020-02-14 20:00:00');
+insert into reservation (number_of_guests, meal_id, created_date) values (10, 4,'2020-02-11 17:00:00');
+
+--review data
+insert into review (title, description, meal_id, stars, created_date) values ('Excellent taste', '', 4, 3,'2020-02-09 00:00:00');
+insert into review (title, description, meal_id, stars, created_date) values ('Delicious!!', 'The pancakes were very very crispy with a flavorful sauce as a dip. Definitely will recommend', 1, 4, '2020-02-01 00:00:00');
+insert into review (title, description, meal_id, stars, created_date) values ('Too soft', 'This bread was indeed very different and was melting in mouth', 3, 4, '2020-02-01 00:00:00');
+
+
+QUERIES
+Meal:
+1. SELECT * FROM meal;
+2. insert into meal (title, description, location, `when`, max_reservations, price, created_date) values ('Chai', 'Indian style tea with milk', 'Fredriskberg', '2020-12-02 22:00:00', 100, 15.50, '2020-11-02 13:13:00');
+3. SELECT * FROM meal WHERE id = 5;
+4. UPDATE meal 
+SET description = 'Yoghurt drink in mango flavor'
+WHERE id = 5;
+5. DELETE from meal
+WHERE id = 6;
+
+Reservation:
+1. SELECT * FROM reservation;
+2. insert into reservation (number_of_guests, meal_id, created_date) values (35, 5,'2020-02-14 17:00:00');
+3. SELECT * FROM reservation WHERE id = 2;
+4. UPDATE reservation 
+SET number_of_guests = 25 
+WHERE id = 4;
+5. DELETE from reservation
+WHERE id = 4;
+
+Review:
+1. SELECT * FROM review;
+2. insert into review (title, description, meal_id, stars, created_date) values ('Too spicy', 4, 1,'2020-01-28 00:00:00');
```

Comment:
Error Code: 1136. Column count doesn't match value count at row 1
insert into review (`title, description, meal_id, stars, created_date`) values (`'Too spicy', 4, 1,'2020-01-28 00:00:00'`);

Reply:
Implemented the feedback